docs/telefonica.css

```diff
@@ -0,0 +1,205 @@
+/*
+
+Uncomment this block is CORS is disabled.
+
+@font-face {
+    font-family: TelefonicaCAP;
+    src: url("https://www.telefonica.com/es/home/telefonica-theme/css/custom/theme/fonts/TelefonicaCAP/otf/TelefonicaCAP.otf"),url("/telefonica-theme/css/custom/theme/fonts/TelefonicaCAP/woff/TelefonicaCAP.woff") format("woff");
```

Comment:
RTD just includes the `telefonica.css` CSS file and renders it as directed. A simple `@import` statement should be fine - you can easily download and render appropriate **Open Source** fonts using the [Google Fonts API](https://fonts.google.com/) -  you could use a nearest open source equivalent if that is acceptable. FIWARE uses Monserrat:

```css
@import url('https://fonts.googleapis.com/css?family=Montserrat%3A100%2C100italic%2C200%2C200italic%2C300%2C300italic%2Cregular%2Citalic%2C500%2C500italic%2C600%2C600italic%2C700%2C700italic%2C800%2C800italic%2C900%2C900italic&ver=4.9.6#038;subset=latin,latin-ext');
```

However I believe the standard Telefonica isn't **open source** so I personally would not like to add it to this PR and expose it directly in a GitHub repository - it would be best to talk to your legal department to see if this would cause any issues.

The commented part of the existing css file **would** load the Telefonica fonts from `www.telefonica.com` **if**  `https://thinking-cities.readthedocs.io` was whitelisted - that would be a cleaner idea if whitelisting is possible and permitted.

As it is, if you  have it could install the appropriate font locally if you have it to see the docs in *Full Telefonica* mode.